src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  background: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    object-fit: contain;
+  }
+
+  &__name {
+    font-weight: 500;
```

Comment:
U've already used it in html {}

Reply:
without this line tests won't pass and title weight will be 400